Fix:

error: call to undeclared function 'isascii'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
error: call to undeclared library function 'islower' with type 'int (int)'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
error: call to undeclared library function 'toupper' with type 'int (int)'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
error: implicit declaration of function 'errx' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'err' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'warnx' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: call to undeclared function 'warnx'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
--- put.c.orig	2001-05-10 12:10:49.000000000 -0400
+++ put.c	2024-02-04 17:33:24.000000000 -0500
@@ -12,9 +12,10 @@
  */
 
 #include <stdio.h>
+#include <err.h>
 #include <string.h>
 #include <assert.h>
-/*#include <ctype.h>*/
+#include <ctype.h>
 #include "config.h"
 
 #include "skey.h"
--- skeyinfo.c.orig	2001-05-10 12:10:49.000000000 -0400
+++ skeyinfo.c	2024-02-04 17:34:03.000000000 -0500
@@ -29,6 +29,8 @@
 
 /*#include <limits.h>*/
 #include <pwd.h>
+#include <err.h>
+#include <ctype.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
--- skeyinit.c.orig	2001-05-10 12:10:49.000000000 -0400
+++ skeyinit.c	2024-02-04 17:29:39.000000000 -0500
@@ -28,6 +28,7 @@
 #include "config.h"
 
 #include <errno.h>
+#include <err.h>
 #include <ctype.h>
 #include <pwd.h>
 #include <stdio.h>
--- skeylogin.c.orig	2001-05-10 12:10:49.000000000 -0400
+++ skeylogin.c	2024-02-04 17:35:19.000000000 -0500
@@ -25,6 +25,7 @@
 #include <sys/types.h>
 
 #include <ctype.h>
+#include <err.h>
 #include <errno.h>
 #include <fcntl.h>
 #include <stdio.h>
